cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
Well, we could. We'd just need to make a copy of the studio base template and translate it to django, like we did with the LMS template.